u_conn.go

```diff
@@ -481,7 +481,51 @@ func (uconn *UConn) GetOutKeystream(length int) ([]byte, error) {
 }
 
 // SetVersCreateState set min and max TLS version in all appropriate places.
-func (uconn *UConn) SetTLSVers(minTLSVers, maxTLSVers uint16) error {
+// If minTLSVers == 0 && minTLSVers == 0,
+// SetTLSVers() will try to parse to parse the version from extensions.
+func (uconn *UConn) SetTLSVers(minTLSVers, maxTLSVers uint16, extensions []TLSExtension) error {
+	if minTLSVers == 0 && maxTLSVers == 0 {
+		// if version is not set explicitly in the ClientHelloSpec, check the SupportedVersions extension
+		supportedVersionsExtensionsPresent := 0
+		for _, e := range uconn.Extensions {
```

Comment:
```suggestion
		for _, e := range extensions {
```
I believe `uconn.Extensions` should be `extensions` here. As is, Chrome 72 fails with the error "tls: server selected unsupported protocol version 304" when it should negotiate TLS 1.3. With the change, Chrome 72 works ok in our test suite. iOS 12.1 also appears to work ok in our test suite.